package.json

```diff
@@ -29,7 +29,7 @@
   },
   "scripts": {
     "generate-readme": "cross-env NODE_ENV=production babel-node ./tools/generate-readme",
-    "build": "webpack --mode production && babel ./src/js/ --ignore '__tests__' --out-dir ./dist --copy-files && cross-env BABEL_ENV=es6 babel ./src/js/ --ignore '__tests__' --out-dir ./dist/es6 --copy-files",
+    "build": "webpack --mode production && babel ./src/js/ --out-dir ./dist --copy-files --ignore '**/**/__tests__' --ignore 'src/**/portal.js' --no-copy-ignored && cross-env BABEL_ENV=es6 babel ./src/js/ --out-dir ./dist/es6 --copy-files --ignore '**/**/__tests__' --ignore 'src/**/portal.js' --no-copy-ignored && rm -r ./dist/__tests__/ ./dist/**/__tests__/ ./dist/**/**/__tests__",
```

Comment:
why do we need 
```
rm -r ./dist/__tests__/ ./dist/**/__tests__/ ./dist/**/**/__tests__",
```
If the tests shouldn't be included in `dist` from the first place.

Reply:
Not sure if it was a bug or something that we were missing but there was no way to ignore/exclude the snapshot files effectively. The `__tests__` directories were still being included but all they had in them were the snapshots so we figured this was the best way to get rid of the snapshot files from the dist folder

Reply:
We shouldn't use the `rm` command.
We should ignore the directory and the files without adding and removing it on the distribution package.

Reply:
Coping our conversation thread for more transparncy:
>The issue seems to be that the --no-copy-ignore flag only applies to js files so the snapshot files are being ignored but the --copy-files flag is still copying them to the dist folder since they're of the form *.js.snap. The workarounds I have found have either used a different package to do it (usually jest-gulp) or use a rm/del command to get rid of the snap files
>Yeah the babel CLI is pretty limited with alot of its options and it doesn't seem like theres any other flags that would be applicable
the docs for the CLI are here https://babeljs.io/docs/en/babel-cli